app/scripts/controllers/app-state.js

```diff
@@ -461,7 +460,7 @@ export default class AppStateController extends EventEmitter {
         this._approvalRequestId,
       );
     } catch (error) {
-      log.error('Failed to accept transaction approval request', error);
+      log.error('Failed to accept approval request', error);
```

Comment:
In the very unlikely event this error occurs, should we be explicit and say `unlock approval request`?